nuxeo-jsf/nuxeo-platform-ui-web/src/main/java/org/nuxeo/ecm/platform/ui/web/binding/DownloadMethodExpression.java

```diff
@@ -88,7 +88,7 @@ public String getExpressionString() {
 
     @Override
     public boolean isLiteralText() {
-        return blobExpression == null ? null : blobExpression.isLiteralText();
+        return blobExpression == null ? false : blobExpression.isLiteralText();
```

Comment:
To be sure, we choose this format:
 ```blobExpression == null ? false : blobExpression.isLiteralText();``` 
regarding this one:
```blobExpression != null && blobExpression.isLiteralText();``` ?

Reply:
Both work. Yes your is cleaner, I'll update.